privacyidea/lib/tokenclass.py

```diff
@@ -205,30 +205,25 @@ def user(self):
         :return: The owner of the token
         :rtype: User object
         """
-        user_object = None
-        realmname = ""
-        if self.token.user_id and self.token.resolver:
-            username = get_username(self.token.user_id, self.token.resolver)
-            rlist = self.token.realm_list
-            # FIXME: What if the token has more than one realm assigned?
-            if len(rlist) == 1:
-                realmname = rlist[0].realm.name
-            if username and realmname:
-                user_object = User(login=username,
-                                   resolver=self.token.resolver,
-                                   realm=realmname)
+        user_object = User()
```

Comment:
Hm, we change the return value of this function in case the token has no owner. Before, we returned ``None``, now we return the empty user object. Is this intentional?

Reply:
I think yes. Because it allows calling functions to handle things better, since they always get a user object.

Reply:
Will do 2019-02-14

Reply:
done in 49f1ac75